docs/product-installation-guide/src/main/asciidoc/clustering-artemis-activate-proc.adoc

```diff
@@ -10,5 +10,5 @@ Elasticsearch installed on the main node of the cluster
 . Add the following properties to the `EAP_HOME/standalone/configuration/standalone.xml` file:
 * Add `<socket-binding name="activemq" port="61616"/>` to the socket-binding-group element.
 * Add `<remote-acceptor name="activemq-acceptor" socket-binding="activemq"/>` in `<server name="default">` of the messaging-activemq subsytem element.
```

Comment:
I'd recommend to provide jboss-cli commands here to set this up, either to replace this or in addition to that